nomad/structs/services.go

```diff
@@ -68,6 +68,7 @@ type ServiceCheck struct {
 	Interval               time.Duration       // Interval of the check
 	Timeout                time.Duration       // Timeout of the response from the check before consul fails the check
 	InitialStatus          string              // Initial status of the check
+	Notes                  string              // Specifies arbitrary information for humans. This is not used by Consul internally
```

Comment:
Do you think we could add a little validation on this field? Maybe we could at least limit the length of notes to 250 characters? There's already [a method](https://github.com/hashicorp/nomad/blob/d4cec2afc484917af27e98531bc43c91be232cf2/nomad/structs/services.go#L415) for validating Consul fields.

Reply:
Good idea, pushed a fix